interfaces/policy/basedeclaration_test.go

```diff
@@ -598,7 +598,6 @@ var (
 		"cups":                    {"app"},
 		"cups-control":            {"app", "core"},
 		"dbus":                    {"app"},
-		"docker-support":          {"core"},
```

Comment:
Why is this removed here?

Reply:
whoops sorry this shouldn't be removed, an artifact of my editor trying to apply a formatter to this file every time I open it and needing to discard the other changes

Reply:
removed this